roles/matrix-riot-web/defaults/main.yml

```diff
@@ -28,3 +28,6 @@ matrix_riot_web_embedded_pages_home_path: ~
 
 # Controls whether the self-check feature should validate SSL certificates.
 matrix_riot_web_self_check_validate_certificates: true
+
+# cross use of variable defined in matrix-synapse role
```

Comment:
I guess we should adjust this comment to say what the setting does.

Mentioning the matrix-synapse role is not necessary, especially given that each role can be used independently (especially in the future - see #80)